src/main.ts

```
@@ -1,10 +1,48 @@
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
+import * as session from 'express-session';
+import * as dotenv from 'dotenv';
 import { ValidationPipe } from '@nestjs/common';
 import { HttpExceptionFilter } from './filters/http-exception.filter';
+import { createClient } from 'redis';
+
+// Use require for connect-redis
+const RedisStoreFactory = require('connect-redis');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)